src/files/files.service.ts

```diff
@@ -1,21 +1,21 @@
-import {HttpException, HttpStatus, Injectable} from '@nestjs/common';
+import { Injectable, InternalServerErrorException } from '@nestjs/common';
 import * as path from 'path';
 import * as fs from 'fs';
 import * as uuid from 'uuid';
 
 @Injectable()
 export class FilesService {
-    async createFile(file): Promise<string> {
-        try {
-            const fileName = uuid.v4() + '.jpg';
-            const filePath = path.resolve(__dirname, '..', 'static')
-            if (!fs.existsSync(filePath)) {
-                fs.mkdirSync(filePath, {recursive: true})
-            }
-            fs.writeFileSync(path.join(filePath, fileName), file.buffer)
-            return fileName;
-        } catch (e) {
-            throw new HttpException('Error when write file', HttpStatus.INTERNAL_SERVER_ERROR)
-        }
+  async createFile(file): Promise<string> {
+    try {
+      const fileName = uuid.v4() + '.jpg';
+      const filePath = path.resolve(__dirname, '..', 'static');
+      if (!fs.existsSync(filePath)) {
+        fs.mkdirSync(filePath, { recursive: true });
+      }
+      fs.writeFileSync(path.join(filePath, fileName), file.buffer);
+      return fileName;
+    } catch (e) {
+      throw new InternalServerErrorException('Error when write file');
```

Comment:
Since we're rethrowing a new error, we have to log original one: otherwise it's impossible to investigate the roots in case of any issues.
```suggestion
      console.log(e);
      throw new InternalServerErrorException('Error when write file');
```